httpcore/_types.py

```diff
@@ -4,8 +4,8 @@
 
 from typing import Dict, List, Optional, Tuple, Union
 
-StrOrBytes = Union[str, bytes]
-Origin = Tuple[bytes, bytes, int]
-URL = Tuple[bytes, bytes, int, bytes]
-Headers = List[Tuple[bytes, bytes]]
-TimeoutDict = Dict[str, Optional[float]]
+StrOrBytesType = Union[str, bytes]
+OriginType = Tuple[bytes, bytes, int]
+URLType = Tuple[bytes, bytes, int, bytes]
+HeadersType = List[Tuple[bytes, bytes]]
+TimeoutDictType = Dict[str, Optional[float]]
```

Comment:
@yeraydiazdiaz Are you okay with these changes? Now that I see it across the codebase the `*Type` suffix looks kinda redundant, ("of course it's a `Type`, it's in an annotation…"), so I'm actually not too sure it's a good idea…

Reply:
I don't have a strong opinion tbh. I guess it is a bit redundant, I'll switch it back.